### PR TITLE
ecosystem consideration: prefixing files with 'go-'

### PR DESCRIPTION
**Pull request by Drakirus**
Default to use a `go` directory instead of `desktop`.
Add a flag, `--path`, to overwrite the go-flutter directory path.

### CODE REVIEW

**Comment by GeertJohan** [comment]
Why? Flutter uses windows/mac/darwin, theres no contlict..

**Comment by GeertJohan** [comment]
I've commented in https://github.com/go-flutter-desktop/go-flutter/issues/209

File: cmd/init.go [hunk 1]
```diff
@@ -53,16 +56,16 @@ var initCmd = &cobra.Command{
...
 
 		wd, err := os.Getwd()
 		if err != nil {
 			fmt.Printf("hover: Failed to get working dir: %v\n", err)
 			os.Exit(1)
 		}
 
-		cmdGoModInit := exec.Command(goBin, "mod", "init", projectPath+"/desktop")
-		cmdGoModInit.Dir = filepath.Join(wd, "desktop")
+		cmdGoModInit := exec.Command(goBin, "mod", "init", projectPath+buildPath)
```
Comment (davidmartos96) [inline cmd/init.go]: Not directly in the project, but I was looking at the PR changes.
I think a `/` is needed before `buildPath` right?

Comment (Drakirus) [reply cmd/init.go]: yes, the "/" is needed.
Thanks!

File: cmd/build.go [hunk 2]
```diff
@@ -34,6 +35,7 @@ func init() {
 	buildCmd.Flags().StringVarP(&buildBranch, "branch", "b", "", "The 'go-flutter' version to use. (@master or @v0.20.0 for example)")
 	buildCmd.Flags().BoolVar(&buildDebug, "debug", false, "Build a debug version of the app.")
 	buildCmd.Flags().StringVarP(&buildCachePath, "cache-path", "", "", "The path that hover uses to cache dependencies such as the Flutter engine .so/.dll (defaults to the standard user cache directory)")
+	buildCmd.Flags().StringVarP(&buildPath, "path", "p", defaultBuildPath, "The path that hover uses to save the 'go-flutter' desktop source code")
```
[the remark this answers is not in the document]
Comment (Drakirus) [reply cmd/build.go]: agree. delete the argument.

File: cmd/common.go [hunk 3]
```diff
@@ -71,9 +71,9 @@ Fail:
 }
 
 func assertHoverInitialized() {
-	_, err := os.Stat("desktop")
+	_, err := os.Stat(buildPath)
 	if os.IsNotExist(err) {
-		fmt.Println("hover: Directory 'desktop' is missing, did you run `hover init` in this project?")
+		fmt.Println("hover: Directory " + buildPath + " is missing, did you run `hover init` in this project?")
```
[the remark this answers is not in the document]
Comment (Drakirus) [reply cmd/common.go]: addeed CLI [Y/n] to migrate

**Pull request merged**